pkg/volume/util.go

```diff
@@ -29,30 +29,63 @@ import (
 	"k8s.io/kubernetes/pkg/watch"
 
 	"github.com/golang/glog"
+	"k8s.io/kubernetes/pkg/api/resource"
 )
 
-// ScrubPodVolumeAndWatchUntilCompletion is intended for use with volume Recyclers.  This function will
+func GetAccessModesAsString(modes []api.PersistentVolumeAccessMode) string {
```

Comment:
This code is nearly identical to [`func GetAccessModesAsString` in `pkg/api/helpers.go`](https://github.com/kubernetes/kubernetes/blob/master/pkg/api/helpers.go#L185).

Why not use that or refactor it to a more suitable location?